backend/api/Controllers/ParcelController.cs

```diff
@@ -137,6 +150,11 @@ public IActionResult AddParcel([FromBody] Model.ParcelModel model)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "To support standardized routes (/update/{id})")]
         public IActionResult UpdateParcel(int id, [FromBody] Model.ParcelModel model)
         {
+            model.Address.Postal = model.Address.Postal.Replace(" ", "");
```

Comment:
Move formatting of postal to service layer instead so that it's handled with any future endpoints.